src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Models/Arkivmelding.cs

```diff
@@ -0,0 +1,398 @@
+using System;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace Altinn.Common.EFormidlingClient.Models
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Arkivmelding"/> class. This class represents the actual arkivmelding.
+    /// This class i autogenerated. XSD definition: https://github.com/difi/felleslosninger/blob/gh-pages/resources/arkivmelding/arkivmelding.xsd
+    /// </summary>
+    [XmlRoot(ElementName = "arkivmelding", Namespace = "http://www.arkivverket.no/standarder/noark5/arkivmelding")]
+    public class Arkivmelding
```

Comment:
Are we able to identify which properties that are required and which that are optional ? Would be a great help to the app developers I believe.

Reply:
Seems like a lot of work. So we'll leave it as is for now.